rpc/src/main/scala/org/bitcoins/rpc/RpcUtil.scala

```diff
@@ -10,13 +10,13 @@ import org.bitcoins.core.util.BitcoinSLogger
 import org.bitcoins.rpc.client.RpcClient
 import org.bitcoins.rpc.config.{ AuthCredentials, DaemonInstance }
 
-import scala.concurrent.{ Await, Future }
 import scala.concurrent.duration.DurationInt
+import scala.concurrent.{ Await, Future, Promise }
 import scala.util.Try
 
-trait TestUtil extends BitcoinSLogger {
+trait RpcUtil extends BitcoinSLogger {
```

Comment:
What i meant here is keeping a `TestUtil`, but moving the timing stuff into a new src file called `RpcUtil`